R/data.R

```diff
@@ -69,7 +109,44 @@
 #' @source vignette_data.R
 "hosp_data"
 
-#' Example hospital admissions dataset for evaluation
+
+
+
+#' Example hospital admissions dataset spatially independent sites.
+#'
+#'  A dataset containing the simulated daily hospital admissions
+#' (labeled here as `daily_hosp_admits`) by date of admission (`date`).
+#'  Additional columns that are required are the population size of the
+#'  population contributing to the hospital admissions. It is assumed that
+#'  the wastewater sites are subsets of this larger population, which
+#'  is in the package data assumed to be from a hypothetical US state.
+#'  The data generated are daily hospital admissions but they could be any other
+#'  epidemiological count dataset e.g. cases. This data should only contain
+#'  hospital admissions that would have been available as of the date that
+#'  the forecast was made. We recommend that users try to format their data
+#'  to match this format.
+#'
+#' This data is generated via the default values in the
+#'  `generate_simulated_data()` function. They represent the bare minumum
+#'  required fields needed to pass to the model, and we recommend that users
+#'  try to format their own data to match this formate.
+#'
+#' The variables are as follows:
+#' \describe{
+#'   \item{date}{Date the hospital admissions occurred, formatte din ISO8601
+#'   standatds as YYYY-MM-DD}
+#'   \item{daily_hosp_admits}{The number of individuals admitted to the
+#'   hospital on that date, available as of the forecast date}
+#'   \item{state_pop}{The number of people contributing to the daily hospital
+#'   admissions}
+#'   }
+#' @source vignette_data.R
+"hosp_data_ind"
+
```

Comment:
I personally think this makes sense for your workflow to save both the spatially independent and dependent datasets as package data, but an alternative approach would just be to generate them directly in the analysis using the `generate_simulated_data()` function and tuning the parameters! And we might want to do that if we are going to vary some of the other parameters at some point (e.g. try different R(t) trajectories for example). Nothing to change now just a comment.